packages/mdc-shape/_mixins.scss

```diff
@@ -20,45 +20,25 @@
 // THE SOFTWARE.
 //
 
-// Sets 45-degree angle corners of a certain size (akin to border-radius but for flat corners).
-@mixin mdc-shape-angled-corner($background-color, $top-left-size, $top-right-size: $top-left-size, $bottom-right-size: $top-left-size, $bottom-left-size: $top-right-size) {
-  $corners: (top, left, $top-left-size),
-    (top, right, $top-right-size),
-    (bottom, right, $bottom-right-size),
-    (bottom, left, $bottom-left-size);
+@import "./variables";
+@import "./functions";
 
-  @each $y, $x, $size in $corners {
-    @if $size > 0 {
-      // Assume that the size passed represents the width/height rather than the diagonal.
-      // This makes it trivial e.g. to accomplish a horizontal arrow cap by specifying exactly half the shape's height.
-      // Unfortunately, Sass doesn't have built-in exponentiation, and node-sass custom functions seem to hang.
-      // (https://github.com/sass/node-sass/issues/857 may be related, but the workaround didn't seem to work.)
-      // Fortunately, if we assume 45 degree cutouts all the time, a^2 + b^2 = c^2 simplifies to a*sqrt(2).
-      $diagonal-length: $size * 1.4142135623730951;
+@mixin mdc-shape-radius($radius, $rtl-reflexive: false) {
+  @if mdc-shape-has-pill-radius($radius) {
+    @error "Invalid radius: '#{$radius}' is not supported for this component";
```

Comment:
How are we supporting pill? Currently it just unconditionally fails here. I'm a little confused since I vaguely recall seeing more to this in previous prototypes.

Reply:
This mixin accepts `$radius` which is absolute value only. Where as resolve-pill mixin accepts radius of absolute and literal. 

Simplified this a bit in my following PR.